javascript/uber_search.js

```diff
@@ -33,7 +33,7 @@ var UberSearch = function(data, options){
   }, options)
 
   var context          = this
-  var view             = $('<span class="uber_select"></span>')
+  var view             = $('<span class="uber_select" id="' + options.wrapperId +'"></span>')
```

Comment:
```suggestion
  var view             = $('<span>', { class: 'uber_select', id: options.wrapperId })
```

Reply:
`wrapperId` existed before. What was it used for and how can we just start using it here too?

Reply:
@njakobsen sorry yes this is just for the container. This should have been added in this commit: https://github.com/culturecode/uber_select/commit/0040d74e81d84b4e353dfd57487b930b54b695fd

Where i depend on it with the index for each list item. It isn't super necessary for UberSelect itself, but it would be nice that if I pass in an ID, i have a handle to the uberselect with the given ID, especially since once we uber_select, we don't really have a handle the the uberSelect object, just the underlying select.